src/iop/basecurve.c

```diff
@@ -186,30 +186,30 @@ static const basecurve_preset_t basecurve_camera_presets[] = {
   // clang-format off
 
   // nikon d750 by Edouard Gomez
-  {"Nikon D750", "NIKON CORPORATION", "NIKON D750", 0, FLT_MAX, {{{{0.000000, 0.000000}, {0.018124, 0.026126}, {0.143357, 0.370145}, {0.330116, 0.730507}, {0.457952, 0.853462}, {0.734950, 0.965061}, {0.904758, 0.985699}, {1.000000, 1.000000}}}, {8}, {m}}, 0, 1},
```

Comment:
all those changes are not correct I think. The autoaplly should be 0 (deactivated) for the  built-in basecurve. The way to autoapply them is to create let the user create a preset based on it and then set the autoapply with possibly conditions from within dt.

Reply:
The logic I used was this:
1. We have generic brand curves
1. We have specific cameras curves.

Both can be auto-applied on user request, the specific curves superceding the generic brand ones. So, if the user requested it and the preset has an auto-apply flag, we auto-apply it, wether it's a specific on a generic preset. If the user requested to not do it, we discard the auto-apply no matter the preset. The thing is the specific presets did not have the auto-apply flag except for the D750. So, in my mind, that just makes the generic and specific presets consistent with each other.